ui/all.js

```diff
@@ -0,0 +1,50 @@
+$(document).ready(function(){
+    const grid = document.querySelector(".grid");
+    let pointer = [0, 0];
+    console.log(pointer[0], pointer[1]);
+
+    // console.log(grid.getAttribute("row", "col"))
+    $('.keyboard').on('click', 'button', function(event) {
+        $(this).parent().parent().find('button').removeClass('activate');
+        $(this).addClass('activate');
+        
+    });
+
+    $('.keyboard').on('click', '.btn', function(event) {
+        if (pointer[1] >= 5) {
+            alert("please press enter or delete!");
```

Comment:
May delete the alert part

Reply:
would delete that after finishing the hint animations